po/common/fr.po

```diff
@@ -4381,7 +4381,7 @@ msgstr "Les photos du produits peuvent aussi être fournies séparément, via la
 
 msgctxt "attribute_group_labels_name"
 msgid "Labels"
-msgstr "Labels"
+msgstr "Étiquettes"
```

Comment:
There seem to be a number of entries that have been translated again, sometimes in wrong ways, like this one.

Reply:
fixed in Crowdin